### PR TITLE
[#90808674] Buttons

### PR DESCRIPTION
**Pull request by ingedmundo**
As a developer I want a CSS class for each button so that I can get the correct format for my CrowdInt's site project.

### CODE REVIEW

**Comment by zalapeach** [comment]
LGTM

File: assets/stylesheets/modules/button.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    -webkit-transition-property: margin, -webkit-transform;
+  }
+  &:hover span:before {
+    -webkit-transform: rotate(45deg);
+    -webkit-transition-delay: 0, .2s;
+    margin-top: 0;
+  }
+  span:after {
+    -webkit-transition-delay: .2s, 0;
+    -webkit-transition-duration: .2s;
```
Comment (ingilniero) [inline assets/stylesheets/modules/button.scss]: Aren't we missing the other vendor prefixes? You could use a mixin for these prefixes. I think SASS have one already, not sure.

**Comment by ingilniero** [comment]
Beside my comment LGTM. Nice work with the button animation.

File: public/stylesheets/modules/button.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+/Users/zalapeach/.rvm/gems/ruby-2.2.0/gems/sass-3.4.13/vendor/listen/lib/listen/listener.rb:252:in `on_change'
+/Users/zalapeach/.rvm/gems/ruby-2.2.0/gems/sass-3.4.13/vendor/listen/lib/listen/listener.rb:290:in `block in initialize_adapter'
+/Users/zalapeach/.rvm/gems/ruby-2.2.0/gems/sass-3.4.13/vendor/listen/lib/listen/adapters/polling.rb:48:in `call'
+/Users/zalapeach/.rvm/gems/ruby-2.2.0/gems/sass-3.4.13/vendor/listen/lib/listen/adapters/polling.rb:48:in `poll_changed_directories'
+/Users/zalapeach/.rvm/gems/ruby-2.2.0/gems/sass-3.4.13/vendor/listen/lib/listen/adapter.rb:299:in `block in start_poller'
+*/
+body:before {
+  white-space: pre;
+  font-family: monospace;
+  content: "Error: Undefined mixin 'buttons'.\A         on line 1 of assets/stylesheets/modules/button.scss, in `buttons'\A         from line 1 of assets/stylesheets/modules/button.scss\A \A 1: .btn-1 { @include buttons(transparent, 5px solid $alto, $silver, $silver-dark, $white) }\A 2: .btn-2 { @include buttons(transparent, 5px solid $white, $white, $white, $silver-dark) }\A 3: .btn-3 { @include buttons($alto, 5px solid $alto, $white, $silver-dark, $white) }"; }
```
Comment (softr8) [inline public/stylesheets/modules/button.css]: Is this a real error ?

Comment (zalapeach) [inline public/stylesheets/modules/button.css]: Yes, it is an error, but these errors is when sass compile the files by separate, and we need the main css file, which contains all the css.

**Pull request merged**